packages/itwinui-css/src/select/select.scss

```diff
@@ -74,6 +74,29 @@
   }
 }
 
+@mixin iui-select-button-borderless {
+  --_iui-input-background-color: transparent;
+  --_iui-input-border-color: transparent;
+  transition: background-color var(--iui-duration-1) ease-out;
+
+  &:where(:hover, :active) {
+    --_iui-input-background-color: var(--iui-color-background-transparent-hover);
+    --_iui-input-border-color: transparent;
+    --_iui-input-with-icon-hover-border-color: transparent;
+  }
+
+  &:focus-visible {
+    outline-offset: -1px;
+    outline-width: 1px;
+  }
+
+  &:is(.iui-disabled, :disabled, [aria-disabled='true']) {
```

Comment:
Did this to match with our pre-existing borderless buttons… however, I think we should discuss this as a team.  

Buttons do not have an empty / placeholder state, so a disabled button with no background color looks fine.
Selects DO have an empty / placeholder state, so the placeholder & disabled states are almost indistinguishable.  Thinking about keeping the default disabled state.